providers/hexonet/auditrecords.go

```diff
@@ -9,16 +9,6 @@ import (
 // supportable by this provider.
 func AuditRecords(records []*models.RecordConfig) error {
 
-	// if err := recordaudit.TxtNoLongStrings(records); err != nil {
-	// 	return err
-	// }
-	// Not needed as of 2021-03-26
-
-	if err := recordaudit.TxtNoMultipleStrings(records); err != nil {
-		return err
-	}
-	// Still needed as of 2021-03-07
```

Comment:
While your in this file, could you update those dates to be `2021-10-03`?  Thanks!